.github/workflows/tests.yml

```diff
@@ -82,8 +75,16 @@ jobs:
       - uses: denoland/setup-deno@v1
         with:
           deno-version: ${{ env.DENO_VERSION }}
-      - run: |
-          curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- -v $WASMEDGE_VERSION
+      - shell: bash
+        run: deno run -A https://raw.github.com/metatypedev/ghjk/feat/mvp/install.ts
+      - shell: bash
+        env: 
```

Comment:
I don't think we can avoid that but it should only be relevant in CI contexts. `BASH_ENV` is only sourced by **non-interactive** bash shells that see it. Interactive shells, unlike the former, load from `.bashrc` most folks don't need to do that. And on dev machines, most non-interactive shells are spawned within interactive ones that are already ghjk activated so I don't see folks relying on this a lot.

As an alternative, folks can just directly `source` the hook within their non-interactive shells. `BASH_ENV` is useful to avoid that call on every shell invocation.

Reply:
Let's move/rename the path such that `$HOME/.local/share/ghjk/env.$SHELL` so we keep it as simple as possible 👍